rust/Cargo.toml

```
@@ -8,9 +8,10 @@ members = [
 resolver = "2"
 
 [workspace.dependencies]
-log = "0.4.8"
 zingolib = { git="https://github.com/zingolabs/zingolib", default-features=true, branch = "stable" }
 zingoconfig = { git="https://github.com/zingolabs/zingolib", default-features=true, branch = "stable" }
+uniffi = { version = "0.27.0" }
```

In general let's allow the minor `X.Y.Z` the ("Z") of semver to auto-update by specifying deps like this:

`dep = { version = "X.Y" }`

This will mean that cargo update is free to select the latest minor version, so we'll automatically get security updates.

---

@AloeareV does using only the first two elements of the semver string make sense to you?

---

I can change this anytime...

---

i agree, also "version" isnt even needed here
```suggestion
uniffi = "0.27"
```